examples/Example/src/showcase/betterSwipeable/AppleStyleSwipeableRow.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useRef } from 'react';
+import { Alert, I18nManager, StyleSheet, Text } from 'react-native';
+import {
+  BetterSwipeable,
+  SwipeableController,
+  RectButton,
+} from 'react-native-gesture-handler';
+import Animated, {
+  Extrapolate,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+
+export const AppleStyleSwipeableRow: React.FunctionComponent<unknown> = (
```

Comment:
```suggestion
export const AppleStyleSwipeableRow = (
```

Check out "Why is `React.FC` discouraged? What about `React.FunctionComponent`/`React.VoidFunctionComponent`?" section
https://github.com/typescript-cheatsheets/react

Reply:
Removed in bc6e272fe973285a8091d03df3dccdfe502bf498